docs/source/introduction/indepth/detailed_about.rst

```diff
@@ -0,0 +1,42 @@
+.. _detailed_about:
+
+Detailed concepts
++++++++++++++++++
+
+Here we present more detailed concepts of HTTomo's framework, such as,
+:ref:`info_sections`, :ref:`info_reslice`, :ref:`info_data` (:ref:`chunks_data`,:ref:`blocks_data`), :ref:`info_wrappers`,
```

Comment:
Looks like rst is a bit finicky, there apparently needs to be a space after a comma and before a `:ref:` otherwise it doesn't render the link correctly:
```suggestion
:ref:`info_sections`, :ref:`info_reslice`, :ref:`info_data` (:ref:`chunks_data`, :ref:`blocks_data`), :ref:`info_wrappers`,
```
See the attached screenshot for the link being displayed incorrectly without the space after the comma:
![link-not-displayed](https://github.com/DiamondLightSource/httomo/assets/13002919/a179a97d-92cd-4bb6-9e41-b43efded70b9)